scipy/optimize/_linprog_util.py

```diff
@@ -1064,11 +1082,6 @@ def vstack(blocks):
     lb_some = np.logical_not(lb_none)
     ub_some = np.logical_not(ub_none)
 
-    # if preprocessing is on, lb == ub can't happen
-    # if preprocessing is off, then it would be best to convert that
-    # to an equality constraint, but it's tricky to make the other
-    # required modifications from inside here.
```

Comment:
Curious why these are removed?

Reply:
Mainly because I think `_get_Abc()` should just convert the problem to standard form. I can see that it is tempting to turn `lb == ub` into an equality constraint here, but that would slip presolve functionality into `_get_Abc()`. I think if one would want to prevent spurious inequalities, one would just enable presolve.